{{ cookiecutter.repo_name }}/OWNERS

```diff
@@ -1,8 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - durandom
-  - tumido
+{%- set applist = cookiecutter.approvers.split(',') %}
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 4: found character that cannot start any token.